tests/consts.go

```diff
@@ -25,13 +25,17 @@ func createBusForTest() gbus.Bus {
 }
 
 func createBusWithOptions(svcName string, deadletter string, txnl, pos bool) gbus.Bus {
+	return createBusWithRetries(svcName, deadletter, txnl, pos, 4, 15)
+}
+
+func createBusWithRetries(svcName string, deadletter string, txnl, pos bool, retries uint, retryDuration int) gbus.Bus {
```

Comment:
last tiny modification. let's turn this into createBusWithConfig that receives a gbus.BusCinfiguration.
That way, as we add configuration options we will not need to create additional factory methods for callers.

Reply:
ok